app/components/ui/atoms/Textarea.js

```diff
@@ -1,19 +1,14 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { th } from '@pubsweet/ui-toolkit'
-
-const Root = styled.div`
-  display: flex;
-  flex-direction: column;
-  max-width: calc(${th('gridUnit')} * 56);
-  margin-bottom: ${th('space.3')};
-`
+import { Flex, Box } from 'grid-styled'
 
 const Label = styled.label`
   font-size: ${th('fontSizeBaseSmall')};
-  display: block;
 `
 
+// TODO - remove when this function is added to pubsweet
```

Comment:
This was added to pubsweet!

Reply:
Pubsweet doesn't have a `Textarea` component This function only exists within their `TextField`.

As @tamlyn has pointed out in the `SubjectAreaDropdown` PR, it would be nice to have this function separated from the `TextField` component & therefore usable as a utility function across multiple components. But I would suggest that this refactor in pubsweet's repo can wait. We can abstract that function out of `TextField` at the same time as we add a `TextArea` to their repo?